test/Image_Tests/src/Data/Matrix_Spec.enso

```diff
@@ -0,0 +1,333 @@
+from Standard.Base import all
+import Standard.Test
+import Standard.Image.Data.Matrix
+
+spec =
```

Comment:
Please name all the tests as in the previous file, and please ensure that you've tested all the failure cases.